tasks/app.js

```diff
@@ -0,0 +1,19 @@
+var endOfLine = require('os').EOL;
+process.stdin.setEncoding('utf8');
+
+process.stdin.on('readable', () => {
+  let initialInput;
+
+  while ((initialInput = process.stdin.read()) !== null) {
+    process.stdout.write(`${reverseString(initialInput) + endOfLine}`);
+  }
+});
+
+process.stdin.on('end', () => {
+  process.stdout.write('end');
+});
+
+function reverseString(string) {
+  var reversedString = string.trim().split('').reverse();
```

Comment:
Consider utilising "let" or "const" instead of "var"s. This is a best practice in modern JS that allows to communicate the purpose of the variable alongside with some V8 optimisations.
In case of this variable consider using "const" since the variable does not change in the current scope.